source/concepts/what-is-ember-js.md

```diff
@@ -124,15 +124,18 @@ You can tell Ember about these dependencies like this:
 
 ```javascript
 var president = Ember.Object.create({
-  firstName: "Barack",
-  lastName: "Obama",
+  init: function() {
+    this._super.apply(this, arguments);
+    this.set('firstName', 'Barack');
```

Comment:
Its a best practice according to [AirBnB](https://github.com/airbnb/javascript).

`Never use arguments, opt to use rest syntax ... instead.`

But also: `Never name a parameter arguments. This will take precedence over the arguments object that is given to every function scope.`